### PR TITLE
Using new Assets client nuget

### PR DESCRIPTION
**Pull request by polonzer**
https://lykkex.atlassian.net/browse/LWDEV-9202

- using the new Assets client nuget;
- changed 'duble' into 'decimal' where possible;
- tiny refactoring;

### CODE REVIEW

File: src/Lykke.Service.HFT/Modules/ServiceModule.cs [hunk 1]
```diff
@@ -65,13 +65,13 @@ protected override void Load(ContainerBuilder builder)
...
 
-            RegisterAssets(builder);
-
             RegisterOrderStates(builder);
 
             BindRabbitMq(builder, currentSettings.HighFrequencyTradingService);
 
-            RegisterPeriodicalHandlers(builder);
+#if !DEBUG
+            RegisterPeriodicalHandlers(builder);
```
Comment (paradise) [inline src/Lykke.Service.HFT/Modules/ServiceModule.cs:73]: is this `if DEBUG` needed?

File: src/Lykke.Service.HFT/Services/StartupManager.cs [hunk 2]
```diff
@@ -47,7 +47,9 @@ public async Task StartAsync()
                 realm.SessionClosed += (sender, args) => { _sessionRepository.TryRemoveSessionId(args.SessionId); };
             }
 
+#if !DEBUG
             await _apiKeyCacheInitializer.InitApiKeyCache();
```
Comment (paradise) [inline src/Lykke.Service.HFT/Services/StartupManager.cs:51]: and this

**Review by paradise — APPROVED** [review]

File: src/Lykke.Service.HFT/Modules/ServiceModule.cs (hunk 1 above)
Comment (polonzer) [reply src/Lykke.Service.HFT/Modules/ServiceModule.cs:73]: This is a long-term process that can influence other instances. I wouldn't like this code to run when I am debugging.

File: src/Lykke.Service.HFT/Services/StartupManager.cs (hunk 2 above)
Comment (polonzer) [reply src/Lykke.Service.HFT/Services/StartupManager.cs:51]: This is a long-term process that can influence other instances. I wouldn't like this code to run when I am debugging.

**Pull request merged**